.github/workflows/postinstall.tests.yml

```diff
@@ -31,8 +31,7 @@ on:
     types: [ opened, reopened, ready_for_review, synchronize ]
 
 env:
-  PYTHON3_VERSION: '3.6'
-  PYTHON2_VERSION: '2.7'
+  PYTHON3_VERSION: '3.8'
```

Comment:
1. Python 3.8 is not available on CentOS 7, which the project still supports. Just Python 3.6

2. Rocky Linux's platform-python is Python 3.6:
   ```shell
   docker run --rm rockylinux:8 /usr/libexec/platform-python --version
   ```

So the minimum supported version still needs to be Python 3.6.

Reply:
The minimum supported version needs to be an unsupported version? SMH Red Hat....